go/test/endtoend/backup/vtbackup/backup_only_test.go

```diff
@@ -234,6 +234,30 @@ func initTablets(t *testing.T, startTablet bool, initShardPrimary bool) {
 		}
 	}
 
+	// wait for both tablet to get into replica state in topo
+	waitUntil := time.Now().Add(10 * time.Second)
```

Comment:
I think 30 seconds is a better timeout for something like this. GitHub actions can at times be super slow, and failing the workflow because it took 15 seconds doesn't really help us (we will have to re-run the entire workflow). 

BUT, `tablet.VttabletProcess.Setup()` waits for the process to start and for the tablet to reach the NOT_SERVING status here (as we didn't specify). So I'm not sure what this additional check is doing here?

Do we think that the tablet is not registered in the topo yet for some reason? Do you have examples of test failures that we think this will resolve? The failures that I had seem seemed to be due to [the transform hook](https://github.com/vitessio/vitess/blob/main/test/vthook-test_backup_transform) never returning. Is that somehow related to that? Did you perhaps re-run that workflow on this PR 10+ times to confirm it was no longer flaky? If it helps, then that's all we really care about. :)

Reply:
I re-tried it and it failed on the 3rd try: https://github.com/vitessio/vitess/runs/8254729459?check_suite_focus=true

It seems like once it fails once, it seems to fail several times in a row. At least that was my experience. So I wonder if it's some state on the Actions host or some other context issue.